actor-sdk/sdk-core-android/android-sdk/src/main/java/im/actor/sdk/BaseActorSDKDelegate.java

```diff
@@ -123,6 +125,17 @@ public MessageHolder getCustomMessageViewHolder(int dataTypeHash, MessagesAdapte
     }
 
     public Uri getNotificationSoundForPeer(Peer peer) {
+        SharedPreferences sharedPreferences = AndroidContext.getContext().getSharedPreferences("notifications", Context.MODE_PRIVATE);
```

Comment:
if you want to use preferences, you'd better use `messenger().getPreferences()`
Anyway, if you'll store notifications settings in prefs they will not sync among clients. Use SettingsModule instead